test/Benchmarks/src/Json.enso

```diff
@@ -1,4 +1,5 @@
 from Standard.Base import all
+import Standard.Base
```

Comment:
Feels better to amend `Base.Json` to just `Json`?

Reply:
The benchmark filename is `Json` so `Json` refers to this file. We can rename the file if needed though.